docs/api/en/objects/LOD.html

```diff
@@ -29,6 +29,7 @@ <h2>Code Example</h2>
 		//Create spheres with 3 levels of detail and create new LOD levels for them
 		for( let i = 0; i < 3; i++ ) {
 			const geometry = new THREE.IcosahedronGeometry( 10, 3 - i );
+			const material = new THREE.MeshBasicMaterial( { color: 0xffff00 } );
```

Comment:
Do you mind moving the material out of the loop (below the `const lod = new THREE.LOD();`)? LODs often share the materials so moving the line ends up with a slightly more optimized setup.